### DISCUSSION TITLE
ebus adapter wiring

### DISCUSSION THREAD

**Discussion by K-Brix** [john30/ebusd#882] [Q&A]
Hello,

Please accept my apologies: this is a newbie question.

I'm new to the ebusd party and am considering an ebus adapter v5 (wifi) for my home heater.
I have a relatively new ebus compatible Saunier Duval DuoCondens (heater/boiler). 
I also have a functionnal HA setup and a MQTT Broker, planning to use the HA ebusd addon and MQTT discovery

The heater has what is, I assume, a RF module connected to the ebus port (photo). The RF module communicate wirelessly with an exacontrol thermostat and an outside temperature probe.

Am I correct to assume that I can parrallel wire the ebus adapter v5 to the RF module and use both of them simultaneously? 
If so, what about programming the heater on a schedule. Will there be a conflict ?

Have a Nice day

JJK

![IMG_6346](https://user-images.githubusercontent.com/96959086/231489169-cad39477-7f5f-471b-bba5-744482ea7a68.jpg)

**Comment by wrongisthenewright** [comment]
I don't' know your specific appliance but the bus is...a bus...so yes you can connect the adapter in parallel with the existing device and tap for data.
If the adapter you're planning to use is powered directly by the bus (no external power supply) then you have to check (by testing it :) ) if the port you're using is able to power simultaneously your adapter and the RF module already connected. In my limited experience it usually works.

Try to find you appliance wiring/technical manual, some boiler have multiple bus ports, in this case is best to use a dedicated port for the adapter.

**Reply by K-Brix** [reply]
Thanks for your answer. I’ll try to find the technical manual. My appliance is very close to Vaillant ones which are well documented in ebusd-configuration. 
I was planning to use an external supply for the ebus-adapter, unless I can find a stable 5v source somewhere on the appliance.

**Reply by debackerl** [reply]
Typically, busses don't like "star" or "tree" topologies, unless the branches are short. For ex, Modbus would allow "studs"/"branches" with a maximal length of 1m from the bus, and you put terminators (resistors) at both ends to avoid reflections. If studs were too long, I think it also generates those reflections issues.

I can't seem to find if this is the case for eBus. It's already different than Modbus in that Modbus requires shielded twisted pair, and I found a Vaillant document stating NOT to use twisted or shielded pairs.
